go/infra/docker/feature-server/Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM golang:1.22.5
 
+ARG SCYLLADB_SHARD_AWARE_ENABLED=False
+ENV SCYLLADB_SHARD_AWARE_ENABLED=$SCYLLADB_SHARD_AWARE_ENABLED
```

Comment:
What's the use of this ENV?

Reply:
Was for optionally selecting the ScyllaDB forked driver, but we're just going to use it by default.